app/actors/curation_concerns/actors/apply_order_actor.rb

```diff
@@ -3,11 +3,28 @@ module Actors
     class ApplyOrderActor < AbstractActor
       def update(attributes)
         ordered_member_ids = attributes.delete(:ordered_member_ids)
+        sync_members(ordered_member_ids)
         apply_order(ordered_member_ids) && next_actor.update(attributes)
       end
 
       private
 
+        def sync_members(ordered_member_ids)
+          return true if ordered_member_ids.nil?
+          existing_members_ids = curation_concern.members.select { |k| k.class.to_s != 'FileSet' }.map(&:id)
```

Comment:
So ordered_member_ids are passed in relative to the children that are being assigned to a work, and then we are trying to synchronize (add any new member ids that are included, and remove any existing member ids that weren't included).. FileSet's wouldn't be included ever from the front-end when dealing with children works. So we wanted to filter out and not compare FileSet's (because they would get removed).. this logic works pretty much the same way the AddWorkActor logic works. Should we chat to clarify?